perun-core/src/test/java/cz/metacentrum/perun/core/entry/VosManagerEntryIntegrationTest.java

```diff
@@ -411,6 +411,21 @@ public void deleteVoWhichNotExists() throws Exception {
 		vosManagerEntry.deleteVo(sess, new Vo());
 	}
 
+	@Test(expected = VoNotExistsException.class)
```

Comment:
This is not needed to change, but I prefer using the `assertThatExceptionOfType()` method from the assertJ library, instead of this JUnit functionality. It is more clear, where the exception should be thrown and you can also easily extend the check with methods that can check, for example, the message of the thrown exception. If you want, you can try to use this method here (without the message check, that was just an example, what can be achieved by using it). :slightly_smiling_face: 

This method is also used in this test file, so you can take a look, how to use it.

Reply:
Thanks for hints! I edit the streams and rewrote the test. :)